spec/services/update_saved_scenario_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe UpdateSavedScenario, type: :service do
+  let(:user) { FactoryBot.create(:user) }
+  let(:result_scenario) { FactoryBot.build(:api_scenario, id: 11) }
+  let(:api_result)  { ServiceResult.success(result_scenario) }
+  let(:result) { described_class.call(saved_scenario, 10) }
+  let!(:saved_scenario) do
+    FactoryBot.create :saved_scenario,
+                      user: user,
+                      id: 648695
+  end
+
+  before do
+    allow(CreateAPIScenario).to receive(:call)
+      .and_return(api_result)
+    allow(saved_scenario).to receive(:scenario)
+      .and_return(ete_scenario_mock)
+
+    saved_scenario.title = nil
+  end
+
+  context 'when the API response is successful' do
+    it 'returns a ServiceResult' do
+      expect(result).to be_a(ServiceResult)
+    end
+
+    it 'is successful' do
+      expect(result).to be_successful
+    end
+
+    describe '#value' do
+      subject { result.value }
+      it {is_expected.to be_a SavedScenario}
+      it {is_expected.to be_persisted}
+    end
+
+    it 'changes the scenario_id on the SavedScenario' do
+      expect{ result }
+        .to change{saved_scenario.scenario_id}
```

Comment:
Apologies for nit-picking over code-style, but could we have a space before and inside blocks? (ditto for a couple of other places in the specs)

e.g. `change { saved_scenario.scenario_id }`